mindsdb/integrations/handlers/box_handler/box_handler.py

```diff
@@ -0,0 +1,238 @@
+import io
+import pandas as pd
+from boxsdk import Client, OAuth2
```

Comment:
@vishwamartur This SDK is under maintenance mode and will be deprecated soon. So, you will have to use `box-sdk-gen`. 
But there's already a PR for box integration with the `box-sdk-gen` sdk. I would suggest you to contribute to other issues.

cc: @chandrevdw31 @ZoranPandovski